Formula/date.rb

```diff
@@ -0,0 +1,46 @@
+class Date < Formula
+  desc "C++11/14/17 library for Date and Time operations based on <chrono> header"
+  homepage "https://github.com/HowardHinnant/date"
+  url "https://github.com/HowardHinnant/date/archive/v2.4.1.tar.gz"
+  sha256 "98907d243397483bd7ad889bf6c66746db0d7d2a39cc9aacc041834c40b65b98"
+
+  option "with-system-tz-db", "Use the operating system's timezone database"
+  option "without-shared", "Build a static version of library"
```

Comment:
No options for new formulas. Please choose the “most common” feature set that users will want, and restrict to that.